runtime/src/message_processor.rs

```diff
@@ -117,7 +117,8 @@ impl MessageProcessor {
             invoke_context.pop();
 
             time.stop();
-            let post_remaining_units = compute_meter.borrow().get_remaining();
+            let post_remaining_units = invoke_context.get_compute_meter().borrow().get_remaining();
```

Comment:
think this is a bug - when `meter` being reset during `invokeContext.push()`, a new refcell<compute_meter> is created, and consumed. Should use this new one for `post`. Doing some testing now

Reply:
Yeah, but not a bug, was not originally meant to be used without the tx-wide feature